styles/_colors.scss

```diff
@@ -2,6 +2,7 @@
 $primary-color: #50e39e !default;
 $primary-color-lighten-1: lighten($primary-color, 7%) !default;
 $primary-color-darken-1: darken($primary-color, 7%) !default;
+$primary-color-darken-2: darken($primary-color, 12%) !default;
```

Comment:
the color isn't define by percent,
prefer this notation => $primary-color-darken-12: darken($primary-color, 12%) !default;  
?